README.md

```diff
@@ -11,31 +11,51 @@ Amazon Simple Email (Amazon SES) library for Delphi applications.
 
 ## Using
 
+### Using with configuration from environment
+
+The better solution is to get configuration from environment variables, and keep the keys out of the code.
```

Comment:
It's not always a better option, so I think we should describe it just as another option instead of a "better solution".